core/light-client/src/mocks/validator_mock_seal.rs

```diff
@@ -0,0 +1,36 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::{error::Error, mocks::validator_mock::ValidatorMock};
+use itp_sgx_io::SealedIO;
+
+/// A seal that return a mock validator.
+#[derive(Clone)]
+pub struct ValidatorMockSeal;
```

Comment:
Mock for the seal, so we can unit test these parts without using a file underneath